### ISSUE TITLE
Sourcemap points to absolute url

### ISSUE THREAD

**Issue by Inviz** [fasiha/mudderjs#23]
Hello Fasiha. I like your library and it's really useful. 

On small thing that is related to build: https://github.com/fasiha/mudderjs/blob/main/dist/mudder.min.mjs.map this file has absolute disk url to the file in `sources` instead of relative path. This makes tools that work with sourcemaps (like merge source maps) fail. Can you please find a way to fix this? thanks.

**Comment by fasiha** [comment]
@Inviz thanks for bringing this to my attention!

Fixed by (1) bumping esbuild and `@chialab/esbuild-plugin-commonjs` to the latest versions, then (2) running `npm audit fix`, and (3) rerunning `npm run compile` to emit new compiler artifacts, including a new [mudder.min.mjs.map](https://github.com/fasiha/mudderjs/blob/main/dist/mudder.min.mjs.map) which uses relative paths. I've published 2.1.1 to npm, let me know if this helps!

Tagging @vitaliytv, in case anything in the above is wrong 🙇!
